ui/component/viewers/videoViewer/internal/videojs.jsx

```diff
@@ -160,18 +160,16 @@ export default React.memo<Props>(function VideoJs(props: Props) {
 
   const isMobile = useIsMobile();
 
-  // will later store the videojs player
   const playerRef = useRef();
   const containerRef = useRef();
-
   const tapToUnmuteRef = useRef();
   const tapToRetryRef = useRef();
-
   const playerServerRef = useRef();
 
   const { url: livestreamVideoUrl } = activeLivestreamForChannel || {};
+  const overrideNativeVhs = !platform.isIPhone();
   const showQualitySelector =
-    !isLivestreamClaim ||
+    (!isLivestreamClaim && overrideNativeVhs) ||
```

Comment:
We can simplify this down now with the new backend...always show quality selector.